tests/test_server.py

```diff
@@ -74,23 +83,31 @@ def test_server_parse_channel_data(server, rtm_start_fixture):
 def test_server_parse_user_data(server, rtm_start_fixture):
     server.parse_user_data(rtm_start_fixture["users"])
     # Find user by Name
-    userbyname = server.users.find('fakeuser')
-    assert type(userbyname) == User
-    assert userbyname == "fakeuser"
-    assert userbyname != "someotheruser"
+    user_by_name = server.users.find('fakeuser')
+    assert type(user_by_name) == User
+    assert user_by_name == "fakeuser"
+    assert user_by_name != "someotheruser"
     # Find user by ID
-    userbyid = server.users.find('U10CX1234')
-    assert type(userbyid) == User
-    assert userbyid == "fakeuser"
-    assert userbyid.email == 'fakeuser@example.com'
+    user_by_id = server.users.find('U10CX1234')
+    assert type(user_by_id) == User
+    assert user_by_id== "fakeuser"
+    assert user_by_id.email == 'fakeuser@example.com'
     # Don't find invalid user
-    userbyid = server.users.find('invaliduser')
-    assert type(userbyid) != User
+    user_by_id = server.users.find('invaliduser')
+    assert type(user_by_id) != User
```

Comment:
okay, its type isn't a user, but what is the type? maybe that's what the assertion should check.

Reply:
Made it check for `None` instead 👍